modmail/bot.py

```diff
@@ -19,51 +16,85 @@ class ModmailBot(commands.Bot):
     """
 
     main_task: asyncio.Task
+    logger = logging.getLogger(__name__)
 
     def __init__(self, **kwargs):
         self.config = CONFIG
         self.internal = INTERNAL
         self.http_session: ClientSession = None
```

Comment:
```suggestion
        self.http_session: t.Optional[ClientSession] = None
```
Do add `import typing as t` at the start of the file.